plugins/woocommerce/client/blocks/.distignore

```diff
@@ -24,14 +24,10 @@
 .wp-env.json
 babel.config.js
 blocks.ini
-composer.json
```

Comment:
.distignore can be obsolete as well as we don;t build any zips for it separately.